tests/test_meshing_utilities.py

```diff
@@ -2247,9 +2247,9 @@ def test_meshing_utilities(new_meshing_session):
         is None
     )
 
-    assert meshing_session.meshing_utilities.delete_marked_faces_in_zones(
-        face_zone_name_pattern="*"
-    ) == [0, 0, 0, 0, 3446]
+    # assert meshing_session.meshing_utilities.delete_marked_faces_in_zones(
```

Comment:
@hpohekar, is this a temporary fix? Or shall we get rid of the commented code altogether?

Reply:
@prmukherj We have kept it for reference even thought it is failing in assertions. You can see that we have already commented other code apart from this.

Reply:
@hpohekar Could you please expand on what you mean by "reference usage" in this context? Thanks.

Reply:
In that case can we update this with a correct code? Or it does not work here? Then anyways the comment will not make sense, right?

Reply:
We had already discussed that we will keep commenting code that gives varying output (in future also). So we are following same rule here.

We should also consider point that same test is passing in each CI run but why it is failing in nightly only.  If we can fix this behavior then we don't need to comment current code.

As already explained above we are keeping commented code for reference that makes sense right ?

Reply:
Exactly that's why to avoid this varying output update every time (as image gets updated on daily basis) we are commenting it.

Reply:
so, later if it is fixed again we will uncomment, right?

Reply:
@hpohekar @mkundu1 @prmukherj 

Some rationale has been given in the above discussion but the rationale is missing from the file itself.

Reply:
@seanpearsonuk Do we want to add reason why we have commented this out ?

![image](https://github.com/user-attachments/assets/ebdcf676-b404-4d4b-9c7e-44152047ab0c)

Reply:
@seanpearsonuk @hpohekar @prmukherj Shall we track this in a github issue? Instead of comparing against fixed values which can change with Fluent version, should we compare with the Scheme call results? The Scheme API should stable as it is documented in Fluent.

Reply:
@mkundu1 Yes, we can do this to avoid this kind of update every time.

Reply:
@mkundu1 I think only concern is about output as it will be unknown if we directly compare with scheme output.